cmd/adm.go

```diff
@@ -167,22 +203,45 @@ func RecreateConfig(cmd *cobra.Command, args []string) error {
 	return config.WriteConfig(*conf, ConfigLocation)
 }
 
-// BashCompletion is the entry point for the `adm completion` cli command
-func BashCompletion(cmd *cobra.Command, args []string) error {
-	err := RootCmd.GenBashCompletionFile("ao.sh")
+// Completion is the entry point for the `adm completion` cli command
+func Completion(cmd *cobra.Command, args []string) error {
+	shell := "bash"
+	if len(args) > 1 {
+		return cmd.Usage()
+	} else if len(args) == 1 {
+		shell = args[0]
+	}
+	var err error
+	filename := ""
+	helpfulInfo := ""
+	switch shell {
+	case "bash":
+		filename = "ao.bash"
+		err = RootCmd.GenBashCompletionFile(filename)
+		helpfulInfo = bashcompletionhelp
+	case "zsh":
+		filename = "ao.zsh"
+		err = RootCmd.GenZshCompletionFile(filename)
```

Comment:
Med bash skrives det ikke ut hjelpetekst, mens zsh skriver ut en kommando med hjelpetekst per linje. Er dette gjort med hensikt? 

For å unngå hjelpetekster så er det GenZshCompletionFileNoDesc() som gjelder.

Reply:
Det finnes såvidt jeg kan se ikke noe alternativ med hjelpetekst for bash.  Jeg gikk for det jeg oppfattet som default og som mest hjelpsomt.  Om det er et mål at det skal oppføre seg så likt som mulig i alle shell, så er det lett å endre.